--- bigloo-4.6a/bmacs/Makefile.orig	2026-05-20 11:40:09.043686469 +0200
+++ bigloo-4.6a/bmacs/Makefile	2026-05-20 11:40:09.045708001 +0200
@@ -174,7 +174,7 @@
 #*---------------------------------------------------------------------*/
 #*    install ...                                                      */
 #*---------------------------------------------------------------------*/
-install: install.$(EMACSBRAND)
+install:
 
 install.emacs21: doinstall
 install.emacs22: doinstall
