--- bigloo2.5b/bdb/Makefile~	Tue Sep  3 09:44:07 2002
+++ bigloo2.5b/bdb/Makefile	Fri Oct 11 20:14:47 2002
@@ -85,9 +85,9 @@
 	   cp $(BOOTBINDIR)/bdb $(BINDIR) && \
 	   chmod $(BMASK) $(BINDIR)/bdb; \
         fi
-	if [ $(FILDIR) != $(BOOTLIBDIR) ]; then \
-	   cp $(BOOTLIBDIR)/bdb.heap $(FILDIR)/bdb.heap && \
-           chmod $(BMASK) $(FILDIR)/bdb.heap; \
+	if [ $(DESTDIR)$(FILDIR) != $(BOOTLIBDIR) ]; then \
+	   cp $(BOOTLIBDIR)/bdb.heap $(DESTDIR)$(FILDIR)/bdb.heap && \
+           chmod $(BMASK) $(DESTDIR)$(FILDIR)/bdb.heap; \
         fi
 	$(MAKE) install-lib LIB=libbigloobdb-$(RELEASE)
 	$(MAKE) install-lnlib LIB=libbigloobdb-$(RELEASE) LN=libbigloobdb_u-$(RELEASE)
